scripts/mongodbintegrationmvp.py

```diff
@@ -33,76 +65,166 @@ def branin(x1, x2):
 ]
 objectives = {obj1_name: ObjectiveProperties(minimize=True)}
 
-# Use Ax's default Sobol trials for 2D problems (aligns with GitHub comment)
+# Use Ax's default Sobol trials for 2D problems
 SOBOL_TRIALS = 5
 
-# Load existing experiment state or initialize new
-record = experiments_col.find_one({"experiment_name": obj1_name})
-if record:
-    saved_trials = record.get("trials", [])
-    n_existing = len(saved_trials)
-    
-    # Calculate remaining Sobol trials: max(target_sobol - existing, 0)
-    remaining_sobol = max(SOBOL_TRIALS - n_existing, 0)
-    
-    if remaining_sobol > 0:
-        generation_strategy = GenerationStrategy([
-            {"model": Models.SOBOL, "num_trials": remaining_sobol},
-            {"model": Models.GPEI, "num_trials": -1}
-        ])
-        print(f"Will run {remaining_sobol} more Sobol trials (have {n_existing} existing)")
-    else:
-        # Remove Sobol step entirely when remaining_sobol = 0
-        generation_strategy = GenerationStrategy([
-            {"model": Models.GPEI, "num_trials": -1}
-        ])
-        print(f"Skipping Sobol (have {n_existing} trials), going to GP")
-    
+def save_ax_snapshot_to_mongodb(ax_client, experiment_name):
```

Comment:
Nice idea on using experiment_name as part of the metadata. Could you update the logic in this script so that it stores all previous snapshots instead of overwriting a single one?